pkg/scheduler/core/generic_scheduler.go

```diff
@@ -344,14 +344,21 @@ func (g *genericScheduler) findNodesThatPassFilters(ctx context.Context, prof *p
 }
 
 func (g *genericScheduler) findNodesThatPassExtenders(pod *v1.Pod, feasibleNodes []*v1.Node, statuses framework.NodeToStatusMap) ([]*v1.Node, error) {
+	var prevFailedButResolvableNodes map[string]*v1.Node
 	for _, extender := range g.extenders {
 		if len(feasibleNodes) == 0 {
 			break
 		}
 		if !extender.IsInterested(pod) {
 			continue
 		}
-		feasibleList, failedMap, err := extender.Filter(pod, feasibleNodes)
+		// We call extenders sequentially, so unfeasible nodes can be excluded
+		// for later extenders. However, failed but resolvable nodes must be
+		// passed as they may be marked unresolvable.
```

Comment:
I doubt if we really want this behavior. Even if we have to do this, isn't in the first round, the _implicit_ infeasible nodes (with Unresolveable code) in statuses should also be included?

Reply:
I agree with Wei, I think we should either pass all nodes to all extenders or pass the feasible ones only. In the framework we are doing the latter, basically we stop on the first filter that returns Unschedulable status: https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/runtime/framework.go#L506

Reply:
sorry for being late (busy on something else)

I've updated the PR, PTAL.

Reply:
created a PR to add a note in the KEP: https://github.com/kubernetes/enhancements/pull/2132/files